postman/schemas/transfer.yml

```diff
@@ -11,6 +11,665 @@ servers:
 security:
 - bearerAuth: []
 paths:
+  "/account-fields":
+    get:
+      tags:
+        - Accounts
+      summary: Get Account Fields
+      description: Get Account Fields
+      operationId: getAccountFields
+      parameters:
+        - name: countryIso3Code
+          in: query
+          required: true
+          description: See [country codes](https://en.wikipedia.org/wiki/ISO_3166-1_alpha-3).
```

Comment:
The description here explains what format to use, but it doesn't explain what this parameter means in the context of the APi call. All fields should have a description explaining what they are used for. 

Ex: The destination country of the transfer for which field requirements are being requested